### PR TITLE
feat: Use explorer api for farms apr

### PR DESCRIPTION
**Pull request by chefjackson**
---

## PR-Codex overview
This PR introduces new functions and updates to improve chain name handling, add APR calculations, and fetch farm information efficiently.

### Detailed summary
- Added `getChainNameInKebabCase` function to `utils.ts`
- Updated `FarmsV3Context` with `apr` field
- Added `chainNamesInKebabCase` and updated `chainNames` in `chainNames.ts`
- Updated `FarmV3ApyButton.tsx` to calculate `globalLpApr`
- Added functions to fetch farms average information
- Updated `usePollFarmsAvgInfo` to fetch farm information efficiently

> The following files were skipped due to too many changes: `scripts/updateLPsAPR.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: scripts/updateLPsAPR.ts [hunk 1]
```diff
@@ -1,223 +1,43 @@
...
-
-  return {
-    stableFarms,
-    normalFarms: [...normalFarms, farm],
-  }
-}
-// ====
-
 const FETCH_CHAIN_ID = [ChainId.BSC, ChainId.ETHEREUM]
 const fetchAndUpdateLPsAPR = async () => {
```
Comment (chef-jojo) [inline scripts/updateLPsAPR.ts:13]: We can remove this daily script later

File: apps/web/src/state/farms/hooks.ts [hunk 2]
```diff
@@ -95,38 +91,28 @@ export const usePollFarmsAvgInfo = (activeFarms: (V3FarmWithoutStakedValue | V2F
 
       const addresses = activeFarms.map((farm) => farm.lpAddress?.toLowerCase())
 
-      const rawResult: any | undefined = await multiQuery(
```
Comment (chef-jojo) [inline apps/web/src/state/farms/hooks.ts:98]: The original query is v3 only? I see there's v3 client and checking

File: apps/web/src/state/farms/hooks.ts [hunk 3]
```diff
@@ -95,38 +91,28 @@ export const usePollFarmsAvgInfo = (activeFarms: (V3FarmWithoutStakedValue | V2F
...
-        return {
-          volumeUSD: averageArray(volumes),
-          tvlUSD: parseFloat(value[0]?.tvlUSD) || 0,
-          feeUSD: averageArray(feeUSDs),
+
+      const info: { [key: string]: { volumeUSD: number; tvlUSD: number; feeUSD: number; apr: number } } = {}
+      for (const addr of addresses) {
+        const farmInfo = farmAvgInfo[addr]
+        info[addr] = {
+          volumeUSD: farmInfo?.volumeUSD7d.decimalPlaces(2).toNumber(),
```
Comment (chef-jojo) [inline apps/web/src/state/farms/hooks.ts:109]: The original value is avg 7d right?

File: scripts/updateLPsAPR.ts (hunk 1 above)
Comment (chefjackson) [reply scripts/updateLPsAPR.ts:13]: yup

Comment (chefjackson) [reply scripts/updateLPsAPR.ts:13]: will remove later on

File: apps/web/src/state/farms/hooks.ts (hunk 2 above)
Comment (chefjackson) [reply apps/web/src/state/farms/hooks.ts:98]: yup

File: apps/web/src/state/farms/hooks.ts (hunk 3 above)
Comment (chefjackson) [reply apps/web/src/state/farms/hooks.ts:100]: yes correct. It was using 7day value to calculate

**Pull request merged**